clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -581,6 +581,157 @@ PointerVariableConstraint::mkString(EnvironmentMap &E,
   return FinalDec;
 }
 
+std::string
+PointerVariableConstraint::mkString2(EnvironmentMap &E,
```

Comment:
Why are you creating this function instead of modifying `mkString`? Furthermore, this function `mkString` is never called.